lib/dom.js

```diff
@@ -3263,3 +3263,25 @@ declare var status: string;
 declare var top: WindowProxy;
 declare function getSelection(): Selection | null;
 declare var customElements: CustomElementRegistry;
+
+/* Notification */
+declare class Notification extends EventTarget {
+  static permission: string;
+  actions: Array<any>;
+  badge: string;
+  body: string;
+  data: string;
+  dir: string;
```

Comment:
I think this is

    dir: 'auto' | 'ltr' | 'rtl';

https://notifications.spec.whatwg.org/#enumdef-notificationdirection